students/kristianjf/lesson02/print_grid.py

```diff
@@ -0,0 +1,58 @@
+'''
+Part 1: Write a function that draws a grid like the following:
++ - - - - + - - - - +
+|         |         |
+|         |         |
+|         |         |
+|         |         |
++ - - - - + - - - - +
+|         |         |
+|         |         |
+|         |         |
+|         |         |
++ - - - - + - - - - +
+'''
+a = '+ '
+b = '- ' * 4
+c = '| '
+d = '  ' * 4
+x = str('{}{}{}{}{}\n').format(a, b, a, b, a)
+y = str('{}{}{}{}{}\n').format(c, d, c, d, c)
+print('Part 1: Write a function that draws a grid\n'+x+y*4+x+y*4+x)
+'''
+Part 2: Make it a function
+Write a function print_grid(n) that takes one integer argument and prints a grid just like before, BUT the size of the grid is given by the argument.
+'''
+def print_grid(x):
+    n = x // 2
+    a = '+ '
+    b = '- ' * n
+    c = '| '
+    d = '  ' * n
+    x = str('{}{}{}{}{}\n').format(a, b, a, b, a)
+    y = str('{}{}{}{}{}\n').format(c, d, c, d, c)
+    print('Part 2: Write a function print_grid(n) that takes one integer argument and prints a grid just like before, BUT the size of the grid is given by the argument.\n'+x+y*n+x+y*n+x)
+
```

Comment:
Small quibble: PEP 8 says that code shouldn't be more than 79 chars in a line. So just be aware that ideally this would be shorter, and that you can break code up with a "\". But, this is an awesome solution!